lib/microos.pm

```diff
@@ -24,8 +24,8 @@ our @EXPORT = qw(microos_reboot microos_login);
 sub microos_login {
     assert_screen 'linux-login-microos', 150;
 
-    # Workers installed using autoyast have no password - bsc#1030876
-    return if get_var('AUTOYAST');
+    # opensuse Workers installed using autoyast have no password - bsc#1030876
```

Comment:
hm , bsc#1030876 is RESOLVED and it looks like since 2018-01-11
And why does `microos` doesn't have a password, you are setting it in line 40 of the autoyast profile :thinking:

Reply:
ah, wait, the autoyast profile is `sle` but you only exclude `opensuse`!? I don't find a opensuse microos autoyast profile, can you point me?

Reply:
Looks like this is a left-over from CASP, cause I can't find any autoyast installation test in O3 for opensuse MicroOS... So, I will remove this line since it's obsolete.